sdk/formrecognizer/ai-form-recognizer/.vscode/settings.json

```diff
@@ -0,0 +1,24 @@
+{
+    "cSpell.enabled": true,
+    "cSpell.ignoreWords": [
```

Comment:
We might want to push some of this file's contents up to https://github.com/Azure/azure-sdk-for-js/blob/main/.vscode/settings.json